modules/nf-core/porechop/porechop/main.nf

```diff
@@ -33,4 +33,16 @@ process PORECHOP_PORECHOP {
         porechop: \$( porechop --version )
     END_VERSIONS
     """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.fastq
+    gzip ${prefix}.fastq
```

Comment:
Not necessary, can just `touch ${prefix).fastq.gz` on the live above

Reply:
Thanks!

Actually, just `touch ${prefix).fastq.gz` (which I tried initially) breaks nf-test as the md5 check on `.gz` files actually tries to compare the file contents. And it therefore requires an actual valid `.gz` file